com.unity.cinemachine/Runtime/Core/CameraState.cs

```diff
@@ -104,7 +104,7 @@ public enum BlendHintValue
         }
 
         /// <summary>
-        /// These hints can be or'ed toether to influence how blending is done, and how state
```

Comment:
This is useful info, why did you delete it?

Reply:
I could put it back yes. or'ed is sorted?

Reply:
No. (A | B) is A and B or'ed together